### DISCUSSION TITLE
Cards in repeater with different actions

### DISCUSSION THREAD

**Discussion by TobiasBorge** [Budibase/budibase#11399] [Help] [answered]
I have a repeater which is connected to a table with 30 rows. The table has different numbers ranging from 1-30 as well as different names.
I have at this point created a screen with all the different cards in a net-format. I want to be able to click on each card and be allocated to a new page. - One individual page for each card. 

How can i proceed to achieve this? Should i maybe not use a repeater?

**Comment by aptkingston** [comment, marked as answer]
I'll convert this to a discussion as this is not a bug.

To answer your question - a repeater is fine. You can reference unique data about each repeated row when defining what actions you'd like to run. For example you could easily navigate to a URL similar to `/row/{{ Repeater.Row._id }}` which would generate a unique URL for each card.

**Reply by melohagan** [reply]
Related docs page: https://docs.budibase.com/docs/url-parameters
